packages/react-core/src/components/Icon/Icon.tsx

```diff
@@ -39,11 +39,7 @@ export const Icon: React.FunctionComponent<IconComponentProps> = ({
   defaultProgressArialabel = 'Loading...',
   ...props
 }: IconComponentProps) => {
-  const _progressIcon = progressIcon ? (
-    progressIcon
-  ) : (
-    <Spinner diameter="1em" isSVG aria-label={defaultProgressArialabel} />
-  );
+  const _progressIcon = progressIcon ? progressIcon : <Spinner diameter="1em" aria-label={defaultProgressArialabel} />;
```

Comment:
```suggestion
  const _progressIcon = progressIcon ?? <Spinner diameter="1em" aria-label={defaultProgressArialabel} />;
```
We may consider cleaning this up with some nullish coalescing